### ISSUE TITLE
SDXL(NoobAI)'s Vpred finetuning wouldn't work… is that expected?

### ISSUE THREAD

**Issue by samliu315** [tdrussell/diffusion-pipe#283]
As the title mentioned, with vpred =true, the resulting checkpoint just generates a messy blob. When it's #vpred =true, the checkpoint only produces noise.
![Image](https://github.com/user-attachments/assets/71a0b6b8-dc75-41d1-8fca-52ca353f3eff)

**Comment by samliu315** [comment]
Even normal SDXL checkpoint fine-tuning only generates noise. Has anyone successfully used diffusion-pipe for SDXL fine-tuning?

**Comment by samliu315** [comment]
I figured it out, just need to lower the learning rate
